clusterloader2/pkg/test/test.go

```diff
@@ -0,0 +1,34 @@
+/*
+Copyright 2018 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package test
+
+import (
+	"fmt"
+
+	"k8s.io/perf-tests/clusterloader2/api"
+	"k8s.io/perf-tests/clusterloader2/pkg/framework"
+)
+
+// RunTest runs test based on provided test configuration.
+func RunTest(f *framework.Framework, conf *api.Config) error {
+	defer f.DeleteAutomanagedNamespaces()
+	err := f.CreateAutomanagedNamespaces(int(conf.AutomanagedNamespaces))
+	if err != nil {
+		return fmt.Errorf("automanaged namespaces creation failed: %v", err)
```

Comment:
I prefer error msgs in active form instead of passive form to make it crisp :)

So here -> "Failed to create automanaged namespaces: %v"

Reply:
I prefer to let it be as it is. In all other files passive form is used.